config/manager/manager.yaml

```diff
@@ -55,5 +56,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.annotations['olm.targetNamespaces']
+        - name: ARGOCD_LABEL_SELECTOR
+          value: ""
```

Comment:
Should we remove `ARGOCD_LABEL_SELECTOR` if it's value is set to empty string here? @raghavi101 

`env.StringFromEnv` in main.go handles empty or missing env value by setting it to default.

Reply:
Cleaned it, thanks @svghadi